person.rb

```diff
@@ -9,6 +9,7 @@ def initialize(_age, name = 'Unknown', parent_permission: true)
     @id = rand
     @name = name
     @parent_permission = parent_permission
+    @rents = []
```

Comment:
Make sure to add this instance variable as `attr_accessor` and define a method for adding rentals. Where the method receives `book`, `date`, and the person itself. Consider doing something like this:

```
def add_rental(book, date)
    Rental.new(date, book, self)
end
```